recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
```

Comment:
Will need to re-render if you are adding this.

Does it not support Python 2 any more?